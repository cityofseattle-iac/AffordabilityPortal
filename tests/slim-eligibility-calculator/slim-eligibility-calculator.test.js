import React from 'react';
import {shallow, mount} from 'enzyme';
import toJson from 'enzyme-to-json';
import SlimEligibilityCalculator from '../../components/calculator/slim-eligibility-calculator';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';

Enzyme.configure({ adapter: new Adapter() }); 

const initialState = {}; 
const mockStore = configureStore();
let wrapper;
let store;
beforeEach(() => {
  store = mockStore(initialState);
 })

describe('SlimEligibilityCalculator UI component', () => {
  it('should render properly', () => {
    const wrapper = shallow(
      <SlimEligibilityCalculator store={store}/>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
