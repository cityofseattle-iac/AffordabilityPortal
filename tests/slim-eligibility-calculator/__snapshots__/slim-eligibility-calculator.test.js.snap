// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SlimEligibilityCalculator UI component should render properly 1`] = `
<ContextProvider
  value={null}
>
  <WithStyles(SlimEligibilityCalculator)
    dispatch={[Function]}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  />
</ContextProvider>
`;
